src/tito/tagger/main.py

```diff
@@ -163,6 +163,12 @@ def _undo(self):
         print
         undo_tag(tag)
 
+    def _changelog_date(self):
+        option = (BUILDCONFIG_SECTION, "changelog_date_with_time")
+        if self.config.has_option(*option) and self.config.getboolean(*option):
+            return strftime("%a %b %d %H:%M:%S %Z %Y")
```

Comment:
This is the correct date format string: `%a %b %d %T %Z %Y`

Reply:
Though your string works too. 👍

Reply:
I've updated it and used `%T`, thanks.